tools/ovc/openvino/tools/ovc/__init__.py

```diff
@@ -18,3 +19,9 @@
 if is_optimum() and optimum_version is not None:
     telemetry = init_ovc_telemetry("Optimum Intel", optimum_version)
     telemetry.send_event("optimum", "import", "import_from_optimum,ov_version:{}".format(get_rt_version()))
+
```

Comment:
I think it is better to rename `is_torch` to `is_torch_compile`. How do you think?

Reply:
it is `nit` and not possible to do right now.